<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InstrumentSetting</class>
 <widget class="QWidget" name="InstrumentSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_instructInfo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>621</width>
     <height>106</height>
    </rect>
   </property>
   <property name="title">
    <string>仪器信息</string>
   </property>
   <widget class="QLabel" name="label_instructID">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>仪   器 ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_number">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>98</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>台站代码:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>98</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>台站名称:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_instructID">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>171</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_number">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>181</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_name">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>501</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_location">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>621</width>
     <height>106</height>
    </rect>
   </property>
   <property name="title">
    <string>位置/时间</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_longitude">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>121</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_longitude">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>经度:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_c">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>高程:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_dimension">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>纬度:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_dimension">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>121</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_altitude">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>81</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>米</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_GPS">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox::indicator {
width: 30px;
height: 30px;
}</string>
    </property>
    <property name="text">
     <string>GPS</string>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_NTP">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>60</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox::indicator {
width: 30px;
height: 30px;
}</string>
    </property>
    <property name="text">
     <string>NTP</string>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_date">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>日期:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_date">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>121</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_time">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>时间:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_time">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>121</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_initialization">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>40</y>
     <width>130</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>软件盘/KeyB</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_saveandexit">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>180</y>
     <width>130</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>退出/Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_instructInfoSet">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>110</y>
     <width>130</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>保存/Save</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>491</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>地震前兆二氧化碳在线分析仪——终端设置</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>231</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>ID：X91XDQYQ1402</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>771</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>参数填写说明</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>591</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>1.仪器ID是指本仪器的ID 代号，仅支持输入数字和英文字母.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>591</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>2.台站代码是指当前仪器的所在的台站代码.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>591</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>3.GPS复选框是指通过GPS模块自动获取仪器所处位置的具体信息。输入框</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>581</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>格式为：经度/纬度：Xddd°mm’ss’    海拔：x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>581</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>4.NTP复选框是指通过时间服务器更新时间。输入框格式为：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>581</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;日期：年-月-日；            时间：时-分-秒；&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>576</x>
      <y>26</y>
      <width>191</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
